data/json/dreams.json

```diff
@@ -563,6 +602,8 @@
     "type": "dream",
     "messages": [
       "You feel smug, knowing that the Queen is dependent on you.",
+      "You relax and enjoy the humming adulation of all your followers.",
```

Comment:
The dream above is about being a worker and this one about being the queen but they're both in the same category?

Reply:
Yeah, but it's a dream, it doesn't have to be logical

besides, a person evolving into an entity with insect like features isn't necessarily going to have exact specific behavior of a given insect species

Reply:
Fris0uman wrote _**"Nitpick but I'm pretty sure no eusocial insect does that, the queen does not really gives order and wouldn't know if an intruder came into the colony."**_

I just wanted to move that comment here, so it can resolved with the other insect dreams.

Reply:
> Yeah, but it's a dream, it doesn't have to be logical

OK, fair enough.

My other comment was just that a queen bee or an ant queen would not send guards gettting rid of an intruder, it barely ever interacts with guards and it would have no idea if an intruder came in the hive because it lives far away from the entrance.  But you can chose to ignore that, maybe mutation dream don't have to be 100% accurate about animal behaviour ^^"